core/arch/arm/kernel/abort.c

```diff
@@ -257,8 +257,12 @@ void abort_print_current_ta(void)
 
 	s->ctx->ops->dump_state(s->ctx);
 
-	if (s->ctx->ops->dump_ftrace)
+#if defined(CFG_TA_FTRACE_SUPPORT)
```

Comment:
Does this belong in this commit?

Reply:
I need to add `s->fbuf = NULL;` to disable syscall trace during before `s->ctx->ops->dump_ftrace(s->ctx);` call. And `s->fbuf` could only be accessed if `CFG_TA_FTRACE_SUPPORT` config option is enabled otherwise it will give compilation error in case of normal build with `ftrace` disabled.

Reply:
OK, makes sense. My confusion comes from the "TA" in the name. Perhaps `CFG_FTRACE_SUPPORT` would be better now that both kernel and user space code can be traced?

Reply:
Shall we take this change in separate PR once this PR is merged? Or would you like to update this PR with this change?

Reply:
I'd say it's better to change the name in this commit, which is precisely where you are introducing a feature that makes the old name a bit ambiguous.

Reply:
Okay, I will update this commit accordingly.